src/api/client/__tests__/AccountManagement.ts

```diff
@@ -47,10 +47,9 @@ describe('AccountManagement class', () => {
     it('should prepare the procedure with the correct arguments and context, and return the resulting transaction', async () => {
       const expectedTransaction = 'someTransaction' as unknown as PolymeshTransaction<void>;
 
-      procedureMockUtils
-        .getPrepareStub()
-        .withArgs({ args: undefined, transformer: undefined }, context)
-        .resolves(expectedTransaction);
+      when(procedureMockUtils.getPrepareStub())
+        .calledWith({ args: undefined, transformer: undefined }, context, {})
```

Comment:
Actually if you go and look at the implementation of `internalRun` method, this is invoked with `{}` which constitutes for the `procedureOpts`. I'm not sure how it worked for sinon maybe it matched the first `n` arguments instead of matching all.